### PR TITLE
Hybrid custody example

### PR DESCRIPTION
**Pull request by nialexsan**
https://www.loom.com/share/1e1c2e5b2e35410fa64e26b514804ba5

## steps to run the project

- clone https://github.com/nialexsan/niftory-samples/tree/hybrid-custody-example
- create an app in [niftory.com](http://niftory.com/)
    - create a project
    - create deploy an NFT smart contract from Niftory
- go to `walletless-onboarding` Niftory example
- put the deployed smart contract address(ex: `8cda917c8f8231bf`) and name(ex: `flowHC` ) into `src/lib/fcl/flow.json`
    
    Example:
    
    ```jsx
    ...		
        "flowHC": {
          "aliases": {
            "testnet": "8cda917c8f8231bf"
          }
        },
    ...
    ```
    
- change `.env` values according to your project:
    
    ```jsx
    NEXT_PUBLIC_API_KEY="<API_KEY>"
    NEXT_PUBLIC_CLIENT_ID="<CLIENT_ID>"
    
    CLIENT_SECRET="<CLIENT_SECRET>"
    ```
    
- run `yarn && yarn dev` to launch the example

## step-by-step process

1. create Niftory account
2. run prerequisite transaction (`Setup filter and factory manager`)
3. run multi-sign transaction to link Niftory child account to a (non-)custodial non-Niftory account (blocto)
4. check linking by fetching NFTs from parent account
5. (optional) delete linking from child or parent account

### CODE REVIEW

**Comment by an1va** [comment]
Looks great @nialexsan. @symetryn can you take a look as well? 

Let's wait for multi-sign before we merge this in to simplify the flow :). We'll try to push that up in the next 2 days 👍

**Comment by nialexsan** [comment]
multi-sign transaction is updated

**Review by an1va — APPROVED** [review]

**Pull request merged**
